pkg/client/k8s/k8s.go

```diff
@@ -960,6 +969,18 @@ func (k *K8sClient) PatchOEV1alpha1SPCAsRaw(name string, patchType types.PatchTy
 	return
 }
 
+// PatchOEV1alpha1CSVAsRaw patches the CSV object with the provided patches
+func (k *K8sClient) PatchOEV1alpha1CSVAsRaw(name, namespace string, patchType types.PatchType, patches []byte) (result *api_oe_v1alpha1.CStorVolume, err error) {
+	result, err = k.oecs.OpenebsV1alpha1().CStorVolumes(namespace).Patch(name, patchType, patches)
+	return
+}
+
+// PatchOEV1alpha1CVRAsRaw patches the CVR object with the provided patches
+func (k *K8sClient) PatchOEV1alpha1CVRAsRaw(name, namespace string, patchType types.PatchType, patches []byte) (result *api_oe_v1alpha1.CStorVolumeReplica, err error) {
```

Comment:
please rename this function as this is does not return bytes. no need to suffix as raw.